Makefile

```diff
@@ -27,6 +27,7 @@ docs:
 
 
 test:
-	docker-compose up -d
```

Comment:
I specified `depends_on` option in docker-compose and it will start Zookeeper/Kafka automatically.
Have you met some situations when Kafka or Zookeeper was not ready before running tests?